src/recipes/savage-banana-soup.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "Savage Banana Soup",
+    "description": "A smoothie that bites back!",
+    "github": "PatrickLohan",
+    "ingredients": [
+        "50g Spinach or Kale",
+        "10g Ground Ginger",
+        "30g Peanut Butter",
+        "10g Ground Flax/Linseed",
+        "200g Banana, ripe",
+        "100g Blueberries",
+        "300ml Almond/Coconut Milk",
```

Extra comma breaking json. Almost there!
```suggestion
        "300ml Almond/Coconut Milk"
```

---

Thanks Alfg, always a stray comma/semi-colon ;)

---

I think I don't need to do anything but click resolve, as you're suggesting changes as maintainer? Pretty new to github, sorry.